pkg/neotest/basic.go

```diff
@@ -29,11 +29,12 @@ import (
 
 // Executor is a wrapper over chain state.
 type Executor struct {
-	Chain         *core.Blockchain
-	Validator     Signer
-	Committee     Signer
-	CommitteeHash util.Uint160
-	Contracts     map[string]*Contract
+	Chain           *core.Blockchain
+	Validator       Signer
+	Committee       Signer
+	CommitteeHash   util.Uint160
+	Contracts       map[string]*Contract
```

Comment:
BTW, this code is unused.

Reply:
Still relevant.

Reply:
You mean to remove field `Contracts`?

Reply:
Yes, this field is not used anywhere. Do we need it?